kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/ZooKeeperHiveClientHelper.java

```diff
@@ -32,12 +32,13 @@ class ZooKeeperHiveClientHelper {
   // Pattern for key1=value1;key2=value2
   private static final Pattern kvPattern = Pattern.compile("([^=;]*)=([^;]*);?");
 
-  private static String getZooKeeperNamespace(JdbcConnectionParams connParams) {
+  public static String getZooKeeperNamespace(JdbcConnectionParams connParams) {
```

Comment:
Can we use smaller range modifier and add `@VisibleForTesting`?

Reply:
done